### PR TITLE
feature(docs): adding textarea component theming docs

### PR DESCRIPTION
**Pull request by Manoj0Marmat**
Signed-off-by: Manoj Marmat <manojrajput762000@gmail.com>

#1094 
Screenshot 2022-10-12 at 1 41 55 AM

### CODE REVIEW

**Comment by Manoj0Marmat** [comment]
@nikolovlazar hello sir i tried to create textarea theme but i'm getting this errors can you help me where i'm messing that i took references from button and input theme please need guidance

File: content/docs/components/textarea/theming.mdx [hunk 1]
```diff
@@ -2,3 +2,37 @@
...
+
+import {
+  App,
+  Index,
+  TextareaTheme,
+} from 'configs/sandpack-contents/component-theming/textarea'
+
+<SandpackEmbed
+  files={{
+    '/theme/components/Button.ts': TextareaTheme,
```
Comment (nikolovlazar) [inline content/docs/components/textarea/theming.mdx:28]: Your `Button.ts` should actually be `Textarea.ts`.

**Review by nikolovlazar — CHANGES_REQUESTED** [review]
@Manoj0Marmat there you go!

File: content/docs/components/textarea/theming.mdx (hunk 1 above)
Comment (Manoj0Marmat) [reply content/docs/components/textarea/theming.mdx:28]: ohh k

Comment (Manoj0Marmat) [reply content/docs/components/textarea/theming.mdx:28]: > Your `Button.ts` should actually be `Textarea.ts`.

working

Comment (Manoj0Marmat) [reply content/docs/components/textarea/theming.mdx:28]: Screenshot 2022-10-12 at 1 59 45 AM
Screenshot 2022-10-12 at 1 59 59 AM
Screenshot 2022-10-12 at 2 00 11 AM

**Comment by Manoj0Marmat** [comment]
@nikolovlazar sorry for all this of messed up i'm beginner

**Pull request closed without merging**